### DISCUSSION TITLE
Frigate still active?

### DISCUSSION THREAD

**Discussion by hobybrenner** [blakeblackshear/frigate#1029] [General]
No major update since February.  This project seems like it has slowed down compared to in the past.  Just curious, as the work so far has been fantastic!

**Comment by blakeblackshear** [comment]
I have just been busy for the past few months with work and personal life.

**Reply by hobybrenner** [reply]
Completely understand!  I just know that from what I have seen there are a few people pretty jazzed about Frigate and I thought I would check in.  Take your time!

**Reply by digiblur** [reply]
Whew...  the stale bot closing everything had me worried too!
